design/ceph/object-store.md

```diff
@@ -12,7 +12,7 @@ A Rook storage cluster must be configured and running in Kubernetes. In this exa
 
 ## Object Store Walkthrough
 
-When the storage admin is ready to create an object storage, he will specify his desired configuration settings in a yaml file such as the following `object-store.yaml`. This example is a simple object store with metadata that is replicated across different hosts, and the data is erasure coded across multiple devices in the cluster.
+When the storage admin is ready to create an object storage, the admin will specify his desired configuration settings in a yaml file such as the following `object-store.yaml`. This example is a simple object store with metadata that is replicated across different hosts, and the data is erasure coded across multiple devices in the cluster.
 ```yaml
 apiVersion: ceph.rook.io/v1alpha1
 kind: ObjectStore
```

Comment:
why not CephObjectStore ?

Reply:
I don't know the history of why a collection of Ceph rgws is called an object-store. For the scope of this design and PR, I want to focus on the other resources. I'd be happy to circle back to the object-store once the work here is done.

Reply:
The CRD was originally named ObjectStore and was later renamed to CephObjectStore. This design doc just pre-dates the renaming.

Reply:
Should I rename the doc? Is there any action to be taken here in this PR?

Reply:
no, that's fine to leave this doc as it is, except perhaps you could rename this to CephObjectStore